app/models/user.rb

```diff
@@ -0,0 +1,8 @@
+class User < ApplicationRecord
+  has_many :posts
+  has_many :comments
+
```

Comment:
- [ ] for the has_many relationship the foreign key should be set.
- [ ] a user has_many likes as well.